### DISCUSSION TITLE
Substituting in strings?

### DISCUSSION THREAD

**Discussion by kotx** [casey/just#949] [Q&A]
I want to do something like this:
```
sourcedir := "src"
builddir  := "dist"

args := "--no-source-map {{sourcedir}}:{{builddir}} --style=compressed"

build:
    sass {{args}}

dev:
    sass --watch {{args}}
```
However the command comes out as `sass --no-source-map {{sourcedir}}:{{builddir}} --style=compressed`, without the substitutions.

**Comment by casey** [comment]
Unfortunately there isn't yet a great way to do this, there's an issue #11, and a pr, which is a little bit stalled, but will probably get picked up and finished eventually: 822.

For now I'd do:

```
args := "--no-source-map " +  sourcedir + ":" + builddir " --style=compressed"
```

Which is a little more verbose, but should work.
